plugins/destination/postgresql/client/migrate.go

```diff
@@ -216,19 +214,22 @@ func (c *Client) createTableIfNotExist(ctx context.Context, table *schema.Table)
 		}
 	}
 
+	pkConstraintName := tableName + "_cqpk"
```

Comment:
shouldn't it actually use the `c.pgTablesToPKConstraints` value if that's available?

Reply:
This is inside `createTableIfNotExist` so there should not be any value

Reply:
right, but what [about](https://pkg.go.dev/github.com/cloudquery/plugin-sdk/v4@v4.5.5/schema#Table) `table.PkConstraintName`?